CHANGELOG.rst

```diff
@@ -2,12 +2,18 @@
 Changelog
 =========
 
+Version 14.4
+============
+
+* Broken example code fixed. `#135 <https://github.com/iqm-finland/cirq-on-iqm/pull/135>`_
+* Allow mid-circuit measurements. `#135 <https://github.com/iqm-finland/cirq-on-iqm/pull/135>`_
```

Comment:
```suggestion
* Allow mid-circuit measurements. `#135 <https://github.com/iqm-finland/cirq-on-iqm/pull/135>`_
* Broken example code fixed. `#135 <https://github.com/iqm-finland/cirq-on-iqm/pull/135>`_
```

My high school English teacher always said to start with the stronger point :D

Reply:
Fixed.